lib/awful/tag.lua

```diff
@@ -1275,19 +1275,23 @@ end
 --- View only a set of tags.
 -- @function awful.tag.viewmore
 -- @param tags A table with tags to view only.
--- @param[opt] screen The screen of the tags.
-function tag.viewmore(tags, screen)
-    screen = get_screen(screen or ascreen.focused())
-    local screen_tags = screen.tags
-    for _, _tag in ipairs(screen_tags) do
-        if not gtable.hasitem(tags, _tag) then
-            _tag.selected = false
-        end
-    end
+function tag.viewmore(tags)
```

Comment:
Please do not remove the `screen` parameter, it breaks the API and would have to wait for 5.0